activiti-cloud-runtime-bundle-service/activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/runtime/ProcessInstanceIT.java

```diff
@@ -666,6 +672,40 @@ void adminShouldDeleteProcessInstance() {
             .isThrownBy(() -> processInstanceRestTemplate.getProcessInstance(processEntity));
     }
 
+    @Test
+    void adminShouldForceDestroyProcessInstance() {
+        //given
+        ResponseEntity<CloudProcessInstance> processEntity = processInstanceRestTemplate.startProcess(
+            processDefinitionIds.get(SIMPLE_PROCESS),
+            null,
+            "business_key"
+        );
+
+        assertThat(processEntity).isNotNull();
+        assertThat(processEntity.getBody()).isNotNull();
+        assertThat(processEntity.getBody().getId()).isNotNull();
+        assertThat(processEntity.getBody().getProcessDefinitionId()).contains("SimpleProcess:");
+
+        //when
+        identityTokenProducer.withTestUser("testadmin");
+        ResponseEntity<CloudProcessInstance> responseEntity = processInstanceRestTemplate.adminDestroy(
+            processEntity,
+            true
+        );
+
+        //then
+        assertThat(responseEntity.getStatusCode()).isEqualTo(HttpStatus.OK);
+
+        assertThatExceptionOfType(AssertionError.class)
+            .isThrownBy(() -> processInstanceRestTemplate.getProcessInstance(processEntity));
```

Comment:
This assertion is a bit weird as it's asserting that there is an assertion error. I guess you are running in this case because there is assertion that is failing inside `getProcessInstance` where the status code is `NOT_FOUND` instead of `OK`. IMO, it's better to rewrite this to make assertion on status code here.

Reply:
This is because `processInstanceRestTemplate.getProcessInstance` method call will throw assertion exception if the response is no `200 OK`. I extended the assert to check for `404 NOT FOUND` error message.